classes/general/featured-content.md

```diff
@@ -0,0 +1,70 @@
+Featured Content - Class
+==========================
+
+Table of Contents
+----------------
+<!-- TOC START min:1 max:3 link:true asterisk:false update:true -->
+  - [ClassID](#classid)
+  - [Explanation](#explanation)
+    - [Protection](#protection)
+  - [CreateClass JSON-schema](#createclass-json-schema)
+  - [Schemas](#schemas)
+  - [Properties](#properties)
+    - [Rome](#rome)
+  - [Entities](#entities)
+<!-- TOC END -->
+
+## ClassID
+`NA`
+
+## Explanation
+To enforce a specific standard of tagging whether or not the content is `featured` or not across all content types.
+
+### Protection
+
+It needs to be clear for the `publisher` whether their content is `featured` or not.
+
+## CreateClass JSON-schema
+```json
+{
+  "name": "Featured Content",
+  "description": "Class for featuring content on the platform."
+}
+```
+
+## Schemas
+
+|Version and Link                                           |   Testnet(s)     |Active|
+|:---------------------------------------------------------:|------------------|:----:|
+| [v0](../../schemas/general/featuredContent0.json)         | `Rome`           | `no` |
+
+## Properties
+### Rome
+##### In Class index 0
```

Comment:
The idea here is to show the `properties` by their `in_class_index`, so these should count from 0-2.

Do you mind fixing that?